### ISSUE TITLE
Phast error on MAF file produced by Ragout: strand must be + for reference sequence

### ISSUE THREAD

**Issue by amizeranschi** [fenderglass/Ragout#79]
Hello,

I am trying to run Phast (the `phyloFit` command) on a MAF file produced by Ragout, but I'm getting the following error:

```
ERROR: bad integers or strand in MAF (strand must be + for reference sequence)
```

And indeed, the referenced line has a "-" for the strand. Is this caused by a bug, or by a design choice in Ragout?

I tried recreating the MAF file from HAL using the `hal2maf` command from halTools, and this time Plast worked correctly.

**Comment by fenderglass** [comment]
I don't think Ragout can generate maf files - do you mind describing how you produced it? Ragout can take hal as input, and will convert it to maf using haltools.

**Comment by amizeranschi** [comment]
Ragout [uses hal2mafMP.py](https://github.com/fenderglass/Ragout/blob/master/ragout/synteny_backend/hal.py#L102) to convert the input HAL file to MAF. I used the resulting file (alignment.maf from the hal-workdir inside ragout's output directory) as input to Phast's phyloFit command and got the error mentioned above.

So, I guess the issue isn't with Ragout itself, but probably with HAL Tools. For the record, I installed packages via Conda/Mamba and acquired HAL Tools from the latest available version of Cactus (which provides HAL Tools) in Bioconda: https://anaconda.org/bioconda/cactus/files

I installed the Conda package version 0.0.2019.03.01, build py27h139dc3e_5 and hal2maf reports version 2.1.

I will post this on the HAL issue tracker.
